pdsfile/pds3file/__init__.py

```diff
@@ -0,0 +1,139 @@
+################################################################################
+# pds3file subpackage & Pds3File subclass with PdsFile as the parent class
+################################################################################
+
+from pdsfile import *
+from . import rules as pdsfile_rules
+
+from pdsfile.general_helper import cache_lifetime_for_class
+
+def cache_lifetime(arg):
+    return cache_lifetime_for_class(arg, Pds3File)
+LOGGER = pdslogger.NullLogger()
+# Initialize the cache
+MEMCACHE_PORT = 0           # default is to use a DictionaryCache instead
+DICTIONARY_CACHE_LIMIT = 200000
+cfg.CACHE = pdscache.DictionaryCache(lifetime=cache_lifetime,
+                                 limit=DICTIONARY_CACHE_LIMIT,
+                                 logger=LOGGER)
+
+class Pds3File(PdsFile):
+    # Override the rules
+    DESCRIPTION_AND_ICON = pdsfile_rules.DESCRIPTION_AND_ICON
+    ASSOCIATIONS = pdsfile_rules.ASSOCIATIONS
+    VERSIONS = pdsfile_rules.VERSIONS
+    INFO_FILE_BASENAMES = pdsfile_rules.INFO_FILE_BASENAMES
+    NEIGHBORS = pdsfile_rules.NEIGHBORS
+    SIBLINGS = pdsfile_rules.SIBLINGS       # just used by Viewmaster right now
+    SORT_KEY = pdsfile_rules.SORT_KEY
+    SPLIT_RULES = pdsfile_rules.SPLIT_RULES
+    VIEW_OPTIONS = pdsfile_rules.VIEW_OPTIONS
+    VIEWABLES = pdsfile_rules.VIEWABLES
+    LID_AFTER_DSID = pdsfile_rules.LID_AFTER_DSID
+    DATA_SET_ID = pdsfile_rules.DATA_SET_ID
+
+    OPUS_TYPE = pdsfile_rules.OPUS_TYPE
+    OPUS_FORMAT = pdsfile_rules.OPUS_FORMAT
+    OPUS_PRODUCTS = pdsfile_rules.OPUS_PRODUCTS
+    OPUS_ID = pdsfile_rules.OPUS_ID
+    OPUS_ID_TO_PRIMARY_LOGICAL_PATH = pdsfile_rules.OPUS_ID_TO_PRIMARY_LOGICAL_PATH
+
+    OPUS_ID_TO_SUBCLASS = pdsfile_rules.OPUS_ID_TO_SUBCLASS
+    FILESPEC_TO_VOLSET = pdsfile_rules.FILESPEC_TO_VOLSET
+    FILESPEC_TO_BUNDLESET = FILESPEC_TO_VOLSET
+
+    BUNDLE_DIR_NAME = 'volumes'
+
+    def __init__(self):
+        super().__init__()
+        # alias for attributes
+        self.volset_  = self.bundleset_
```

Comment:
Can't you also do this for the properties below?

Reply:
When I tried to do the same thing for property, it actually won't work, please see the ipython output (`is_volset` and `is_bundleset` should output the same, but they didn't):
```

In [1]: import pdsfile.pds3file as pds3file
   ...: import pdsfile.pds4file as pds4file
   ...: from pdsfile.preload_and_cache import use_shelves_only
   ...: use_shelves_only(pds3file.Pds3File, True)
   ...: use_shelves_only(pds4file.Pds4File, True)
   ...: pds3file.Pds3File.preload('/Users/yu-jenchang/Dropbox (SETI Institute)/Shared-OPUS/pdsdata/holdings')
   ...: pds4file.Pds4File.preload('/Users/yu-jenchang/Dropbox (SETI Institute)/Pds4FileTest/pds4-holdings')

In [2]: 
   ...: a3 = pds3file.Pds3File.from_abspath('/Users/yu-jenchang/Dropbox (SETI Institute)/Shared-OPUS/pdsdata/holdings/volum
   ...: es/COVIMS_0xxx/COVIMS_0001/data/1999017T031657_1999175T202056/v1308946681_1_002.qub')

In [3]: a3.is_bundleset
Out[3]: False

In [4]: a3.is_volset
Out[4]: ''

In [5]: exit
```